### PR TITLE
Ping boilerplate

### PR DESCRIPTION
**Pull request by QuentinBisson**
Towards https://github.com/giantswarm/giantswarm/issues/23238

Splitting of https://github.com/giantswarm/heartbeatctl/pull/3 part 2

### CODE REVIEW

**Comment by QuentinBisson** [comment]
@TheoBrigitte the comment is resolved here :)

File: cmd/ping.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+	if err != nil {
+		log.Fatalf("Failed to init OpsGenie client: %v\n", err)
+	}
+	c := ctl.NewCtl(repo)
+	pings, err := c.Ping(opts.selectorOptions.ToConfig())
+	if err != nil {
+		log.Fatalf("Failed to ping heartbeats: %v\n", err)
+	}
+	for hbn, ping := range pings {
+		fmt.Printf("heartbeat \"%s\": %s\n", hbn, ping.Message)
```
Comment (TheoBrigitte) [inline cmd/ping.go:114]: `hbn` ?? This is the heartbeat codename right ? If yes I would prefer something like name


```suggestion
	for name, ping := range pings {
		fmt.Printf("heartbeat \"%s\": %s\n", name, ping.Message)
```

File: pkg/ctl/ports.go [hunk 2]
```diff
@@ -19,4 +19,9 @@ type Port interface {
 	// in this case must specify at least one selector or name (to target all
 	// heartbeats specify a `nameExpressions=['.*']` rule explicitly).
 	Disable(*SelectorConfig) ([]heartbeat.HeartbeatInfo, error)
+
+	// Ping pings all heartbeats selected by given SelectorConfig, which
+	// in this case must specify at least one selector or name (to target all
+	// heartbeats specify a `nameExpressions=['.*']` rule explicitly).
```
Comment (TheoBrigitte) [inline pkg/ctl/ports.go:25]: Please add a line on what is returned, especially what value is use as key in the map.

**Review by TheoBrigitte — CHANGES_REQUESTED** [review]

File: pkg/ctl/ports.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (QuentinBisson) [reply pkg/ctl/ports.go:25]: @TheoBrigitte does that work ?

[the remark this answers is not in the document]
Comment (TheoBrigitte) [reply pkg/ctl/ports.go:25]: yes

**Pull request merged**